examples/prometheus_integration/bootstrap_db.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
```

Comment:
@prateek would wiring this test within semaphore be overkill. Essentially it's testing Prometheus with M3DB as a remote storage node.

Reply:
Only concern with adding is impact on build times. Can you see how much time build takes if you add to the existing "Docker Integration Test" job?

Reply:
Currently the "Docker Integration Test" takes around 4 and half minutes to fully run. The longest test we have is around 10 minutes.

Reply:
@prateek sorry for the delay, with both the docker integration test and the prometheus test the [build](https://semaphoreci.com/m3db/m3db/branches/nerd0-issue-727/builds/8) clocks in around 7 minutes which is still under the 10 minutes builds. What do you think?

Reply:
Nice! I'm good with adding it if you are :)